src/views/header/RightHeaderBlock.js

```diff
@@ -78,7 +96,7 @@ const View = boneView.extend({
   // scrollLeft triggers a reflow of the whole area (even only get)
   _sendScrollEvent: function() {
     if (!this.blockEvents) {
-      this.g.zoomer.set("_alignmentScrollLeft", this.el.scrollLeft, {origin: "header"});
+      this.g.zoomer.set("_alignmentScrollLeft", this.getView('headers').el.scrollLeft, {origin: "header"});
```

Comment:
We can no longer scroll the entire top-level container.
This is because canvases are fixed in a place, and the rendered content is translated by the rendering logic itself.